Book-A-Desk.Domain/Book-A-Desk.Domain.fsproj

```diff
@@ -1,14 +1,21 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
     <PropertyGroup>
-        <OutputType>Exe</OutputType>
         <TargetFramework>netcoreapp3.1</TargetFramework>
         <RootNamespace>Book_A_Desk.Domain</RootNamespace>
     </PropertyGroup>
 
     <ItemGroup>
-        <Compile Include="Domain.fs" />
-        <Compile Include="Program.fs" />
+        <Compile Include="Domain.Common.fs" />
+        <Compile Include="Reservation.Domain.fs" />
+        <Compile Include="Reservation.Events.fs" />
+        <Compile Include="Reservation.Commands.fs" />
+        <Compile Include="Reservation.Aggregate.fs" />
+        <Compile Include="Reservation.Command.BookADesk.fs" />
```

Comment:
Can we not use folders in F#? It seems like all of these would make sense to be inside a single folder.

Reply:
We can - I just not used them at the moment to let people see what happened (and in the past some code editors had problems with F# folders and I did not check if that is still the case)